rhg_compute_tools/kubernetes.py

```diff
@@ -209,6 +219,17 @@ def get_cluster(
     if name is not None:
         container["image"] = name
 
+    if tag is not None:
+        container["image"] = re.sub(
+            (
+                r"^(([a-zA-Z0-9_\-\.]+(:[a-zA-Z0-9_\-\.]+)?/)?[a-zA-Z0-9_\-\.]+(/[a-zA-Z0-9_\-\.]+)*)"
+                r"(:[^/]+(@.*)?)?"
+                r"$"
+            ),
+            r"\1",
+            name
```

Comment:
I'm pretty bad at regex but will `name` be None here? b/c if `tag` is defined, than `name` has to be none? should this be `container["image"]` instead of `name`?